scripts/compose.py

```diff
@@ -1051,6 +1050,17 @@ def __init__(self, **options):
                 self.environment["ELASTICSEARCH_PASSWORD"] = "changeme"
                 self.environment["ELASTICSEARCH_USERNAME"] = "kibana_system_user"
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
+        self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
+            "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+
+    @classmethod
+    def add_arguments(cls, parser):
+        parser.add_argument(
+            "--kibana-elasticsearch-url",
```

Comment:
```suggestion
            "--kibana-elasticsearch-urls",
```

as stated in https://github.com/elastic/apm-integration-testing/pull/557#discussion_r321641939 ?

Reply:
it will be incorrect because `--kibana-elasticsearch-url` will set only one URL, but can appear multiple times, it how it was implemented all over the place.

`compose.py 7.2 start --kibana-elasticsearch-url RUL1 --kibana-elasticsearch-url URL2`

Reply:
aha!! That's the missing bit I didn't know. Thanks for the clarification :)